seatunnel-connectors-v2/connector-google-sheets/src/main/java/org/apache/seatunnel/connectors/seatunnel/google/sheets/deserialize/GoogleSheetsDeserializer.java

```diff
@@ -50,7 +52,8 @@ public SeaTunnelRow deserializeRow(List<Object> row) {
             String rowStr = objectMapper.writeValueAsString(map);
             return deserializationSchema.deserialize(rowStr.getBytes());
         } catch (IOException e) {
-            throw new RuntimeException("Object json deserialization exception.", e);
+            throw new GoogleSheetsConnectorException(GoogleSheetsConnectorErrorCode.OBJECT_JSON_DESERIALIZATION_FAILD,
```

Comment:
`CommonErrorCode.JSON_OPERATION_FAILED` is better

Reply:
makes sense, done